.github/workflows/zxc-build-library.yaml

```diff
@@ -111,7 +111,7 @@ jobs:
       #     run: tree /F build
 
       - name: Start the local node
-        run: sudo npx @hashgraph/hedera-local start -d --network local
+        run: sudo npx @hashgraph/hedera-local start -d --network local --network-tag=0.49.0-alpha.3
```

Comment:
I believe `0.49.0` is tagged now? Should we just use that?

Reply:
I can't find a `0.49.0` tag. Can you provide a link? Otherwise this is a version we know is stable with the `local-node`

Reply:
My fault, was looking at protobufs not services